src/BinSkim.Rules/PERules/BA2022.SignSecurely.cs

```diff
@@ -368,7 +393,7 @@ private CryptoError GetSignerHashAlgorithms(IntPtr hWVTStateData, out string has
             return cryptoError;
         }
 
-        private Native.WINTRUST_DATA InitializeWinTrustDataStruct(string filePath, WinTrustDataKind kind, uint signatureIndex = 0)
+        private Native.WINTRUST_DATA InitializeWinTrustDataStruct(string filePath, WinTrustDataKind kind, uint signatureIndex = 0, string pszOID = Native.szOID_CERT_STRONG_SIGN_OS_1)
```

Comment:
@shaopeng-gh For some reason I am not seeing the file changes here. From what I understand, a passing test will have `pszOID == ...KEY_OS_1` and a failing test will have `pszOID == ...SIGN_OS_1`. Is that correct?

Reply:
The additional parameter i added, get passed to the `pszOID` of type `CERT_STRONG_SIGN_PARA` of `signatureSettings.pCryptoPolicy` of `winTrustData`, 
it manipulates how strict the verification WinTrustverify will do.